staging/src/k8s.io/kubectl/pkg/cmd/apply/apply.go

```diff
@@ -386,6 +386,16 @@ func (o *ApplyOptions) Run() error {
 	if len(infos) == 0 {
 		return fmt.Errorf("no objects passed to apply")
 	}
+
```

Comment:
You should revert the changes in this file. As we're introducing the new flag we're also adding these warning where they should be. thanks.